docs/future-work/project-roadmap.md

```diff
@@ -9,35 +9,46 @@ We have a high-level overview table of tasks on our roadmap in our [project plan
 ![Screenshot of our overview sheet. For an up-to-date view, just go to the sheet https://docs.google.com/spreadsheets/d/1HlKqhMzYWxpW6_D0LtDd5qEuWRYHl6xMfxbIZ76lzng/edit?usp=sharing](../figures/future-work/monty_tasks.png)
 
 
+Here is an interactive view of that table, that can be filtered and sorted to find tasks that match your scope appetite, skills and interests.f
+
+[block:embed]
+{
+  "html": false,
+  "url": "https://futureworkwidget.thousandbrains.ai/index.html?columns=title,estimated-scope,rfc,status,skills",
+  "provider": "futureworkwidget.thousandbrains.ai",
+  "href": "https://futureworkwidget.thousandbrains.ai/index.html?columns=title,estimated-scope,rfc,status,skills",
+  "typeOfEmbed": "iframe",
+  "height": "800px",
+  "iframe": true
+}
+[/block]
+
 
 These are tasks where we have a rough idea of how we want to achieve them but we haven't necessarily scheduled when we will work on them or who will work on them. **We welcome anyone who would like to pick up one of these tasks and contribute to it.**
 
 Tasks are categorized in two ways:
 
-- **Which part of Monty does it touch?** This is represented in the columns, and is organized into Sensor Module, Learning Module, Motor System, Voting, Hierarchy, Environment, Framework, and Open-Source/Communication. 
-- **Which capabilitie(s) does it improve?** This is represented by hashtags (#) at the end of the task description. We keep track of the progress on the capabilities in the column on the right. Each task should improve at least one capability but can contribute to multiple.
+- **Which part of Monty does it touch?** This is represented in the columns, and is organized into Sensor Module, Learning Module, Motor System, Voting, Hierarchy, Environment, Framework, and Open-Source/Communication.
```

Comment:
Is this still referring to the spreadsheet? If not, I think the wording needs to be updated (e.g. "This is represented in the columns,..." isn't accurate for the interactive table. Why was the first bullet point removed? Don't we still capture this with the "Metric" field?